src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Security.TransportSecurity.Tests.csproj

```diff
@@ -39,6 +39,10 @@
             <Project>{9098B41C-9C2E-4FC6-B7D8-FC3411736A22}</Project>
             <Name>ScenarioTests.Common</Name>
         </ProjectReference>
+    	<ProjectReference Include="$(WcfInfrastructureCommonProj)">
+      	    <Project>{AFD69665-89A3-42AE-A32E-AB2CBBE6EE7E}</Project>
+            <Name>Infrastructure.Common</Name>
+        </ProjectReference>
```

Comment:
This new project reference is a hint the ToF wrapper project living in TFS will also need to be updated to do the same thing.  That would require a TFS checkin separate from this one, and it could probably be done first to without harm.